### PR TITLE
Extension list UI improvements

### PR DESCRIPTION
**Pull request by blackcoffeexbt**
Improvements to the UI of the extensions list
--
+ Extensions are sorted alphabetically
+ Added a search input field

### CODE REVIEW

File: lnbits/core/static/js/extensions.js [hunk 1]
```diff
@@ -1,4 +1,42 @@
...
+    this.filteredExtensions = this.g.extensions
+  },
+  watch: {
+    searchTerm(term) {
+      // Reset the filter
+      this.filteredExtensions = this.g.extensions
+      if (term !== '') {
+        // Filter the extensions list
+        function extensionNameContains(searchTerm) {
+          console.log('wordToCompare', searchTerm)
```
Comment (motorina0) [inline lnbits/core/static/js/extensions.js:20]: - `console` should be removed

File: lnbits/core/static/js/extensions.js [hunk 2]
```diff
@@ -1,4 +1,42 @@
...
+          return function (extension) {
+            if (
+              extension.name.toLowerCase().indexOf(searchTerm.toLowerCase()) >
```
Comment (motorina0) [inline lnbits/core/static/js/extensions.js:23]: - nitpick: use `String.includes()` instead of `.indexOf()`

**Comment by motorina0** [comment]
LGTM

**Comment by callebtc** [comment]
Yalla! LGTM

**Pull request merged**